### PR TITLE
allow configuring statsd via ENV vars [RUN-103]

### PR DESCRIPTION
**Pull request by jonmoter**
/cc @zendesk/runway 

### References
 - Jira link: https://zendesk.atlassian.net/browse/RUN-103

### Risks
 - None

### CODE REVIEW

File: config/initializers/statsd.rb [hunk 1]
```diff
@@ -1,13 +1,31 @@
 require 'statsd'
 
-config_file = Rails.root.join('config/statsd.yml')
-raise "No such file: config/statsd.yml" unless File.exist?(config_file)
+module StatsdLoader
+  def self.create
+    config = config_from_env || config_from_file
+    raise('Could not initialize statsd by file or by ENV') if config.nil?
```
Comment (sandlerr) [inline config/initializers/statsd.rb]: you could also do `config = config_from_env || config_from_file || raise( ... )` but probably shouldn't

**Comment by sandlerr** [comment]
:+1: although I kinda prefer making everything inherit from zendesk_configuration if possible

**Comment by jonmoter** [comment]
@sandlerr, zendesk_configuration isn't open-sourced, whereas Samson is. We don't want to add that dependency.

File: config/initializers/statsd.rb (hunk 1 above)
Comment (jonmoter) [inline config/initializers/statsd.rb]: Yeah, I thought of that, but it didn't look/feel quite right.

**Pull request merged**